README.md

```diff
@@ -6,33 +6,25 @@ Markdown viewer, editor, and help components for the [Zooniverse](https://www.zo
 
 Available on [npm](http://npmjs.com), include as a dependency using `npm install --save markdownz`
 
-For the MarkdownHelp component, any default styles can be added to `src/css/markdown-help.css`. Don't forget to add a `require` statement for the css file!
-
-This package contains three publicly accessible components: a Markdown viewer and a Markdown editor for Zooniverse-flavored Markdown, and a MarkdownHelp component that displays a modal illustrating styles and how to implement them.
+This package contains three publicly accessible components: a Markdown viewer and a Markdown editor for Zooniverse-flavored Markdown, and a MarkdownHelp component.
 
 Viewer:
 
-```jsx
```

Comment:
Removing this from the readme messed up the code blocks (which makes sense) - Adding again as `javascript`, but let me know if you have something else in mind.